File: dynamicallyTyped/python2/python-practice/codingBat/first_half.py
#return first half of a string

def first_half(str):
  str = raw_input("> ")
  if len(str) >= 2:
    length = len(str)
    half = str[0:length/2]
    print half
  else:
    print "please input a string of length >=2  "
first_half("filler")

File: java/codingBat/string1/firstHalf.java
//Given a string of even length, return the first half. 
//So the string "WooHoo" yields "Woo". 


public class firstHalf{
	public static void main(String[] args){
		System.out.println(firstHalf("WooHoo"));
		System.out.println(firstHalf("HelloThere"));
	}

	public static String firstHalf(String str){
		int l = str.length()/2;
		String half = str.substring(0,l);
		return half;	
	}
}
